--- Makefile.orig	2014-11-20 14:08:26.000000000 +0000
+++ Makefile	2014-11-20 14:08:32.000000000 +0000
@@ -58,7 +58,6 @@
 	install -D ./scripts/vif $(DESTDIR)/$(LIBEXECDIR)/vif
 	install -D ./scripts/vif-real $(DESTDIR)/$(LIBEXECDIR)/vif-real
 	install -D ./scripts/block $(DESTDIR)/$(LIBEXECDIR)/block
-	install -D ./scripts/xen-backend.rules $(DESTDIR)/etc/udev/rules.d/xen-backend.rules
 	install -D ./scripts/tap $(DESTDIR)/$(LIBEXECDIR)/tap
 	install -D ./scripts/qemu-dm-wrapper $(DESTDIR)/$(LIBEXECDIR)/qemu-dm-wrapper
 	install -D ./scripts/qemu-vif-script $(DESTDIR)/$(LIBEXECDIR)/qemu-vif-script
@@ -83,7 +82,6 @@
 	rm -f $(DESTDIR)/$(LIBEXECDIR)/vif
 	rm -f $(DESTDIR)/$(LIBEXECDIR)/vif-real
 	rm -f $(DESTDIR)/$(LIBEXECDIR)/block
-	rm -f $(DESTDIR)/etc/udev/rules.d/xen-backend.rules
 	rm -f $(DESTDIR)/$(LIBEXECDIR)/tap
 	rm -f $(DESTDIR)/$(LIBEXECDIR)/qemu-dm-wrapper
 	rm -f $(DESTDIR)/$(LIBEXECDIR)/qemu-vif-script
